client/mycart/src/app/admin/admin-routing.module.ts

```diff
@@ -11,7 +11,7 @@ const routes: Routes = [
     path: '', component: AdminLayoutComponent, children: [
       { path: 'product/create', component: AddProductComponent },
       { path: 'feedback/view', component: ViewFeedbackComponent },
-      { path: 'product/update', component:UpdateProductComponent}
+      { path: 'product/update', component:UpdateProductComponent},
```

Comment:
Did you miss the product id? Also, format the code.

Reply:
Done. Product ID is auto generated, so not updating it.